code/game/objects/items/storage/bags.dm

```diff
@@ -127,6 +127,11 @@
 	STR.max_w_class = WEIGHT_CLASS_HUGE
 	STR.max_combined_stack_amount = 50
 
+/obj/item/storage/bag/ore/Destroy()
```

Comment:
Destroy is for handling refs and stuff, you dont want to empty contents on it no, stuff like rocks or tablets do it and its super messy and looks shit

Reply:
okay, ill figure something else out